std_msgs/Header header
int32 activity_present 
int32 activity_absent 
int32 ubd_tp
int32 ubd_fn
int32 ubd_fp
int32 ubd_tn
int32 cd_tp
int32 cd_fn
int32 cd_fp
int32 cd_tn
